### PR TITLE
Fix targeted scan regressions

### PR DESCRIPTION
**Pull request by davidegirardi**
Summary:
 - Removed unneeded white-spaces to make ISPF happy
 - Removed the comments about "0 0 READ WRITE EXCEPTION", it's either something that applies only to v1 or probably something related to my routing config
 - Got back scans for a single port and port ranges
 - Fixed the output on non-verbose non-debug runs

### CODE REVIEW

File: PORTSCAN [hunk 1]
```diff
@@ -176,6 +187,7 @@ DO FOREVER
   if excp_sock \= "" then do
     if debug then call OutDebug "EXCEPTION SOCKETS:" excp_sock
     Do i = 1 to WORDS(excp_sock)
+     read_sockets.0 = read_sockets.0 + 1
```
Comment (mainframed) [inline PORTSCAN]: Why is this here?

**Review by mainframed — COMMENTED** [review]
Other than line 190 looks good.

File: PORTSCAN (hunk 1 above)
Comment (davidegirardi) [reply PORTSCAN]: Wrong copy paste. I updated my repo, new PR incoming soon.

**Comment by davidegirardi** [comment]
Closing this, another coming soon.
